src/utils/request/pruneGetRequestBody.ts

```diff
@@ -1,21 +0,0 @@
-import { ServiceWorkerIncomingRequest } from '../../setupWorker/glossary'
-import { isStringEqual } from '../internal/isStringEqual'
-
-type Input = Pick<ServiceWorkerIncomingRequest, 'method' | 'body'>
-
-/**
- * Ensures that an empty GET request body is always represented as `undefined`.
- */
-export function pruneGetRequestBody(
```

Comment:
I think we still perform this logic in MockedRequest. May I please ask you to double-check?